scene/gui/rich_text_label.cpp

```diff
@@ -449,9 +447,7 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 					fh = line_ascent + line_descent;
 
 					if (end && c[end - 1] == ' ') {
-						if (p_mode == PROCESS_CACHE) {
-							spaces_size += font->get_char_size(' ').width;
-						} else if (align == ALIGN_FILL) {
```

Comment:
Might be a bug that this was set but unused (as in, maybe it's meant to be used). It's no longer in the `master` branch, but that code was refactored.